src/LanguageLine.php

```diff
@@ -41,6 +41,13 @@ public static function getTranslationsForGroup(string $locale, string $group): a
                 })
                 ->pluck('text', 'key')
                 ->toArray();
+
+            $result = [];
```

Comment:
Often I prefer using a collection over loops with a temporary value like that. Could you rewrite this piece of code so that only a single collection chain is used?